change/@azure-msal-common-3cdc9d22-67a6-41c7-8d7c-6b42b43c8106.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Do we need this in v2 also?

Reply:
I will open a separate PR for this change in v2.